### ISSUE TITLE
using the bulk endpoint

### ISSUE THREAD

**Issue by lavie** [MauriceButler/bunyan-loggly#8]
Is there any particular reason why the bulk endpoint isn't used when sending buffered log records?
https://github.com/MauriceButler/bunyan-loggly/blob/master/index.js#L41
From what I gather, the `log` function could be invoked by passing an array of log records, instead of being called in a loop, and that would take advantage of bulk posting and the network overhead.

**Comment by MauriceButler** [comment]
Hi @assaflavie 

I do not remember why, I think there may have been an issue, possibly to do with tags, but that would have been some time ago...

I'll test your pull request #9 and if everything is all good get it merged ASAP.

**Comment by lavie** [comment]
Awesome.
Let me know if you want me to make any changes.

Off topic: I've also implemented a somewhat more esoteric feature -
throttling/rate-limit on calls to loggly within a give batch period. You
can see it on my fork as "feature/throttle". If you think that's something
others might find useful, I'd be happy to work on a PR for that.
(The motivation: we integrated with loggly recently and by mistake flooded
our account quota on day 1 - so this serves as a protection against runaway
logs)

Assaf
